grafana/server.sls

```diff
@@ -69,6 +77,20 @@ grafana_{{ theme_name }}_css_override:
 
 {%- endfor %}
 
+{{server.path.data}}:
```

Comment:
See this init.d https://github.com/grafana/grafana/blob/master/packaging/deb/init.d/grafana-server#L80 do still need the default file updated, otherwise path used are the default ones. In systemd, like Ubuntu 16, those directories are not created at all, see https://github.com/grafana/grafana/blob/master/packaging/deb/systemd/grafana-server.service